osu.Game.Tests/Gameplay/TestSceneStoryboardSamples.cs

```diff
@@ -116,7 +116,8 @@ public void TestSamplePlaybackWithRateMods(Type expectedMod, double expectedRate
             AddAssert("sample playback rate matches mod rates", () => sample.Channel.AggregateFrequency.Value == expectedRate);
         }
 
-        private class TestSkin : LegacySkin
+        // TODO: adding IBeatmapSkin to keep changes as minimal as possible, but this shouldn't exist or should be reworked to inherit LegacyBeatmapSkin
```

Comment:
I see no reason for this todo. it's test code. it doesn't need to be 100% pretty.